CONTRIBUTING.md

```diff
@@ -230,7 +230,7 @@ The goal is to make the Uptime Kuma installation as easy as installing a mobile
 
 ## Tools
 
-- [`Node.js`](https://nodejs.org/) >= 14
+- [`Node.js`](https://nodejs.org/) >= 18
 - [`npm`](https://www.npmjs.com/) >= 8.5
```

Comment:
```suggestion
- [`npm`](https://www.npmjs.com/) >= 10.2.4
```

Reply:
We are currently using `lockfileVersion: 3`. I think as long as people are using an npm version that generates this version of lockfile, it should be fine. From the docs it became the default starting from v9.

https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json#handling-old-lockfiles